bin/router/src/pipeline/mod.rs

```diff
@@ -116,13 +119,35 @@ pub async fn execute_pipeline(
         &parser_payload,
     )
     .await?;
-    let query = Cow::Owned(execution_request.query.clone());
+    let query: Cow<'_, str> = Cow::Owned(execution_request.query.clone());
     let variable_payload =
         coerce_request_variables(req, supergraph, execution_request, &normalize_payload)?;
 
     let query_plan_cancellation_token =
         CancellationToken::with_timeout(shared_state.router_config.query_planner.timeout);
 
+    let progressive_override_ctx =
+        request_override_context(&shared_state.override_labels_evaluator, || {
+            ClientRequestDetails {
+                method: req.method().clone(),
+                url: req.uri().clone(),
+                headers: req.headers(),
+                operation: OperationDetails {
+                    name: normalize_payload.operation_for_plan.name.clone(),
+                    kind: match normalize_payload.operation_for_plan.operation_kind {
+                        Some(OperationKind::Query) => "query",
+                        Some(OperationKind::Mutation) => "mutation",
+                        Some(OperationKind::Subscription) => "subscription",
+                        None => "query",
+                    },
+                    query: query.clone(),
+                },
+            }
+        })
+        .map_err(|error| {
+            req.new_pipeline_error(PipelineErrorVariant::LabelEvaluationError(error))
```

Comment:
I wonder if a failure in evaluating the label needs to be marked as a overall request failure? 

I think in such case, the override just needs to be skipped and logged?